app/assets/javascripts/Components/Result/file_viewer.jsx

```diff
@@ -14,10 +14,21 @@ export class FileViewer extends React.Component {
 
   constructor(props) {
     super(props);
+
+    this.max_sizes = {
+      any: 15e6,
```

Comment:
I don't think this global limit is needed, but you can create add a `_default` key to always retrieve a numerical value from this map. In the size check below, only check against one number (the type's corresponding max size, if found, or `_default` if not found).